artifactory/commands/transferfiles/localgeneratedfilter_test.go

```diff
@@ -99,3 +100,20 @@ func TestFilterLocallyGeneratedEnabled(t *testing.T) {
 	assert.False(t, NewLocallyGenerated(context.Background(), servicesManager, "7.54.5").IsEnabled())
 	assert.False(t, NewLocallyGenerated(context.Background(), servicesManager, "6.0.0").IsEnabled())
 }
+
+var getPathInRepoCases = []struct {
```

Comment:
This variable can be made a local variable inside the `TestGetPathInRepo` function below.

Reply:
We agreed to keep it outside the function to make the test short and clear